pytorch_lightning/trainer/connectors/checkpoint_connector.py

```diff
@@ -122,6 +130,9 @@ def restore(self, checkpoint_path: str, on_gpu: bool):
         # load training state (affects trainer only)
         self.restore_training_state(checkpoint)
 
+        log.info(f'Model restored from: {checkpoint_path}')
```

Comment:
Will this be logged on all processes with DDP ?

Reply:
I could cause too much logging, thunks! (though I have no confidence because I am DDP newbie.)  

I think that Notifying which checkpoint is used is very helpful.  
Are there any good hook point to it?

Reply:
Fixed.